### PR TITLE
Fix EZP-24632: VersionInfo not containing all translated names

### PR DESCRIPTION
**Pull request by andrerom**
https://jira.ez.no/browse/EZP-24632

This issue was introduced in cf921fba1a13487d2480ca779d9b7417928b46e9 (#1321).

Instead of just removing the language filter, this change removes name
data from the query, instead it loads it separately to avoid large
result-sets being returned when loading Content and VersionInfo.

### CODE REVIEW

**Comment by andrerom** [comment]
Review ping @pspanja @flovntp @yannickroger

File: eZ/Publish/Core/Persistence/Legacy/Content/Gateway/DoctrineDatabase/QueryBuilder.php [hunk 1]
```diff
@@ -213,13 +217,10 @@ public function createVersionInfoFindQuery()
...
             $this->dbHandler->aliasedColumn($query, 'name', 'ezcontentobject'),
-            $this->dbHandler->aliasedColumn($query, 'language_mask', 'ezcontentobject'),
-            // Content object names
-            $this->dbHandler->aliasedColumn($query, 'name', 'ezcontentobject_name'),
-            $this->dbHandler->aliasedColumn($query, 'content_translation', 'ezcontentobject_name')
+            $this->dbHandler->aliasedColumn($query, 'language_mask', 'ezcontentobject')
         )->from(
             $this->dbHandler->quoteTable('ezcontentobject_version')
-        )->leftJoin(
+        )->innerJoin(
```
Comment (andrerom) [inline eZ/Publish/Core/Persistence/Legacy/Content/Gateway/DoctrineDatabase/QueryBuilder.php]: @pspanja I didn't see any reason why this needs to be leftJoin, hence the change here.

Comment (pspanja) [inline eZ/Publish/Core/Persistence/Legacy/Content/Gateway/DoctrineDatabase/QueryBuilder.php]: +1

**Comment by pspanja** [comment]
Good catch. Looks good to me, +1.

**Pull request merged**